src/Symfony/Component/HttpKernel/Tests/Profiler/ProfilerTest.php

```diff
@@ -61,16 +61,12 @@ public function testFindWorksWithInvalidDates()
 
     protected function setUp()
     {
-        if (!class_exists('SQLite3') && (!class_exists('PDO') || !in_array('sqlite', \PDO::getAvailableDrivers()))) {
-            $this->markTestSkipped('This test requires SQLite support in your environment');
-        }
-
         $this->tmp = tempnam(sys_get_temp_dir(), 'sf2_profiler');
         if (file_exists($this->tmp)) {
             @unlink($this->tmp);
         }
 
-        $this->storage = new SqliteProfilerStorage('sqlite:'.$this->tmp);
+        $this->storage = new FileProfilerStorage('file:'.$this->tmp);
```

Comment:
changing the storage used in tests may be done in older branches too (avoiding the need to skip the tests in some cases, as the file storage is always available)